components/raftstore/src/store/fsm/peer.rs

```diff
@@ -986,6 +986,26 @@ where
         syncer.report_for_self(self_report);
     }
 
+    // Check if the region is in the flashback state.
+    fn on_check_flashback_state(&mut self, ch: Sender<bool>) {
+        let mut is_in_flashback = self.fsm.peer.is_in_flashback;
+        // Read the region local state to check whether the region is in the flashback
```

Comment:
don't need, memory state should be always the latest state

Reply:
Since the memory state is set by `on_ready_result`, it's possible that there is an interval between when apply completes and when the memory state is set 🤔.

Reply:
The ready result must be handled before your flashback proposal is handled. Messages to one region's mailbox are serialized

Reply:
Oh, I get it.